frontend/components/Buttons.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="btncontainer">
+    <button class="resetfilters" @click="resetButtons">
+      <i class="far fa-eye"></i>
+      Zresetuj filtry
+    </button>
+    <button class="saveasfile" href="www.the100clth.pl">
+      <i class="fas fa-cloud-download-alt"></i>
+      Zapisz do pliku
+    </button>
```

Comment:
Also you can use `v-bind="$attrs"`, it will allow you to use Vue methods on 'top level' components, as in example I mentioned before:

```js
<BaseButton @click="resetFilters" :hasIcon="true" :icon="fas fa-cloud">Zresetuj filtry</BaseButton>
```

Reply:
Okay, for now we only have 2 custom buttons, other are provided by Tailwind UI components, I will create an Issue regarding this task.